src/components/Quiz/Quiz.js

```diff
@@ -1,28 +1,118 @@
 import './Quiz.css';
 import { NavLink } from 'react-router-dom/cjs/react-router-dom.min';
+import getCocktails from '../../api-calls';
+import { useState } from 'react';
+import listIngredients from '../../helper-functions';
 
 function Quiz() {
-    const spirits = ['vodka', 'gin', 'rum', 'whiskey', 'tequila', 'scotch', 'cordial', 'sparkling', 'non-alcoholic']
-    const ingredients = ['lime', 'lemon', 'coffee', 'soda', 'cranberry', 'orange', 'grapefruit', 'bitters', 'ginger']
-    const glassware = ['rocks', 'flute', 'hurricane', 'coupe', 'highball', 'shot', 'wine', 'martini']
+
+    const [cocktailResults, setCocktailResults] = useState([]);
+    const [questionNum, setQuestionNum] = useState(0);
+    const [quizError, setQuizError] = useState('');
+    const [cocktail, setCocktail] = useState({});
+
+    const spirits = ['Vodka', 'Gin', 'Rum', 'Bourbon', 'Tequila', 'Scotch'];
+    const ingredients = ['Lime', 'Lemon', 'Coffee', 'Cranberry_juice', 'Orange_juice', 'Grapefruit_juice', 'Bitters', 'Ginger'];
+    const glassware = ['Cocktail_glass', 'Champagne_flute', 'Hurricane_glass', 'Whiskey_sour_glass', 'Highball_glass', 'Shot_glass', 'Collins_glass', 'Martini_glass'];
 
     const makeButtons = (category) => {
-        return category.map(item => <button>{item}</button>)
+        return category.map((item, index) => <button name={item} onClick={(event) => fetchSelection(event.target.name)} key={index}>{item}</button>);
```

Comment:
If category is the string of spirits, ingredients, or glassware, is it possible to extend this map even further and store the `<h2>` in there as well?  `category.map((item, index) => <>  <h2>{Pick A ${item}} </h2> <button></button> </>)`.   Capitalization might be weird though.  You could even go so far as to transform it into a variable that just maps through those three categories and produces the three chunks of JSX, if the categories are always going to be the same!

Reply:
I ended up re-doing the text for each and am going to have to redo the way the categories are set up so that multiple fetches can happen for some buttons but that would be a great way to dry it up otherwise!